toolchain/gcc/Config.version

```diff
@@ -7,13 +7,19 @@ config GCC_VERSION_8
 	default y if arc
 	bool
 
+config GCC_VERSION_9
+	default y if GCC_USE_VERSION_9
+	bool
+
 config GCC_USE_EMBEDDED_PATH_REMAP
 	default y if GCC_VERSION_8
+	default y if GCC_VERSION_9
```

Comment:
I suggest using `default y if ( GCC_VERSION_8 || GCC_VERSION_9 )` to combine these two lines

Reply:
Done!